src/Microsoft.AspNet.Mvc/MvcOptionsSetup.cs

```diff
@@ -67,6 +67,9 @@ public static void ConfigureMvcOptions(MvcOptions options)
             options.ValidationExcludeFilters.Add(typeof(XObject));
             options.ValidationExcludeFilters.Add(typeof(Type));
             options.ValidationExcludeFilters.Add(typeof(JToken));
+            options.ValidationExcludeFilters.Add(typeof(System.Threading.CancellationToken));
+            options.ValidationExcludeFilters.Add(typeof(Http.IFormFile));
+            options.ValidationExcludeFilters.Add(typeof(Http.IFormCollection));
             options.ValidationExcludeFilters.Add(typeFullName: "System.Xml.XmlNode");
```

Comment:
Might want to add a comment here to the effect of - any 'known' types that we bind should be marked as excluded from validation